internal/gitlab/gitlab.go

```diff
@@ -482,15 +482,15 @@ func HydrateGitlabMetaphorRepo(dryRun bool) {
 			return
 		}
 
-		metaphorTemplateDir := fmt.Sprintf("%s/metaphor", cfg.K1FolderPath)
+		metaphorTemplateDir := fmt.Sprintf("%s/k1-app", cfg.K1FolderPath)
 
 		url := "https://github.com/kubefirst/metaphor-template"
 
 		metaphorTemplateRepo, err := git.PlainClone(metaphorTemplateDir, false, &git.CloneOptions{
 			URL: url,
 		})
 		if err != nil {
-			log.Panicf("error cloning metaphor-template repo")
+			log.Panicf("error cloning k1-app-template repo")
 		}
 		viper.Set("create.gitlabmetaphor.cloned", true)
 
```

Comment:
Are we renaming it?

Reply:
I removed the function, it wasnt been used. Had some git issues, I moved it to here https://github.com/kubefirst/kubefirst/pull/209